src/com/google/javascript/jscomp/ProcessCommonJSModules.java

```diff
@@ -244,16 +244,15 @@ private void visitScript(NodeTraversal t, Node script) {
     private void processExports(Node script, String moduleName) {
       if (hasOneTopLevelModuleExportAssign()) {
         // One top-level assign: transform to
-        // /** @const */ var moduleName = rhs
+        // /** @const */ moduleName = rhs
         Node ref = moduleExportRefs.get(0);
         Node newName = IR.name(moduleName);
         newName.putProp(Node.ORIGINALNAME_PROP, ref.getQualifiedName());
 
-        Node newVar = IR.var(newName)
-            .copyInformationFromForTree(ref.getParent());
         Node rhsValue = ref.getNext().detachFromParent();
-        newVar.getFirstChild().addChildToFront(rhsValue);
-        newVar.setJSDocInfo(NodeUtil.createConstantJsDoc());
+        Node newExprResult = IR.exprResult(IR.assign(newName, rhsValue)
+          .copyInformationFromForTree(ref.getParent()));
+        newExprResult.setJSDocInfo(NodeUtil.createConstantJsDoc());
```

Comment:
Please remove this line. <code>@const</code> for variables only makes sense with VAR, not at other assignments.